tests/recipes/test_full_finetune_distributed.py

```diff
@@ -60,9 +63,17 @@ def _fetch_expected_loss_values(self, model_type):
             ("llama3/8B_full", "llama3", "tune", "NO_SHARD"),
         ],
     )
+    @pytest.mark.parametrize("optim_in_bwd", [True, False])
```

Comment:
Currently add one more param input "optim_in_bwd" to have separate test, shall we have the test in another way? @ebsmothers

Reply:
I think this way is OK